drivers/Cargo.toml

```diff
@@ -16,6 +16,7 @@ caliptra-image-types.workspace = true
 caliptra-lms-types.workspace = true
 caliptra-registers.workspace = true
 cfg-if.workspace = true
+sha2.workspace = true
```

Comment:
Can this be a dev dependency? (Or see suggestion below to move sw impl outside of the driver lib)

Reply:
I've migrated it into the target, so this is addressed by the next push.